Source/ThirdParty/libwebrtc/Source/webrtc/webkit_sdk/WebKit/CMBaseObjectSPI.h

```diff
@@ -83,18 +89,20 @@ struct CMBaseProtocolTable {
 
 #pragma pack(push, 4)
 typedef struct {
-   CMBaseClassVersion version;
-   size_t derivedStorageSize;
-
-   Boolean (*equal)(CMBaseObjectRef, CMBaseObjectRef);
-   OSStatus (*invalidate)(CMBaseObjectRef);
-   void (*finalize)(CMBaseObjectRef);
-   CFStringRef (*copyDebugDescription)(CMBaseObjectRef);
-
-   CMBaseObjectCopyPropertyFunction copyProperty;
-   CMBaseObjectSetPropertyFunction setProperty;
-   OSStatus (*notificationBarrier)(CMBaseObjectRef);
-   const CMBaseProtocolTable *protocolTable;
+    CMBaseClassVersion version;
```

Comment:
those are 3rd party source file. Please don't modify them, it makes resync too complicated

Reply:
I can leave the three-space indentation, but I need to add the last two fields below for the fix.

Reply:
I updated the PR to use `WEBRTC_WEBKIT_BUILD`, which is how @youennf prefers code changes to be annotated.